src/main.c

```diff
@@ -7,7 +7,7 @@
 #include <signal.h>
 #include <producer.h>
 #include <utils/helper.h>
-
+#include <version.h>
```

Comment:
Nitpick: Why not just add the three lines here, rather than a new file?
It's slightly counterintuitive in that it gives the impression of an autogenerated file but I have no strong opinions against it.

Reply:
Makes sense.